plyer/platforms/linux/wifi.py

```diff
@@ -17,7 +17,21 @@
 
 class LinuxWifi(Wifi):
     names = {}
-
+    
+    com = Popen(["nmcli", "-t","connection", "show", "--active","--order","active" ], stdout=PIPE )
+    com = com.communicate()[0].strip().split("\n")
+    for i in com:
```

Comment:
This isn't good approach. Logic such as this should be container within a function e.g. a constructor (`__init__` in python). Refactor it please.